app/frontend/src/utils/nominatim.ts

```diff
@@ -0,0 +1,43 @@
+export interface NominatimPlace {
+  address: {
+    [city: string]: string;
+    state_district: string;
+    state: string;
+    postcode: string;
+    country: string;
+    country_code: string;
+  };
+  boundingbox: Array<string>;
+  category: string;
+  display_name: string;
+  icon: string;
+  importance: number;
+  lat: string;
+  licence: string;
+  lon: string;
+  osm_type: string;
+  osm_id: string;
+  place_id: number;
+  place_rank: number;
+  type: string;
+}
+
+export const simplifyPlaceDisplayName = (place: NominatimPlace) => {
+  const addressFields = [
+    "neighbourhood",
+    "suburb",
+    "city",
+    "country",
+    "town",
+    "village",
```

Comment:
```suggestion
    "town",
    "village",
    "neighbourhood",
    "suburb",
    "city",
    "state",
    "country",
```
How about this? Town and village should be first, and in many places like the US and Australia including the state is important. I just hope it is omitted automatically by nominatim where appropriate😓

Reply:
Are villages smaller than towns? Maybe that should be swapped.

Reply:
From what I know villages are smaller ;) Will swap it ;)

Reply:
Should the city be in front of neighbourhood & suburb?

Reply:
Cities are bigger imo